code/exercisesIntro.tex

```diff
@@ -90,9 +90,10 @@
     \small
     \begin{verbatim}
 
-wget http://cern.ch/sponce/C++Course/code.tgz
-tar xzf code.tgz
-cd code
+wget https://github.com/hsf-training/cpluspluscourse/archive\
+/refs/heads/master.zip
+unzip master.zip
```

Comment:
Do people have `unzip` available? Is there a better alternative?

Reply:
People should know how to unzip stuff.

This, however, will be the full course. Let me find out if you can create stable links (== still work when we replace the PDF with a new version) on Indico.

Reply:
It's a 1.2MiB file. I don't think it harms to also download the tex files in addition to the exercises. Also there is no direct support in GitHub to download a subfolder.

With external services, it's possible: https://download-directory.github.io/?url=https%3A%2F%2Fgithub.com%2Fhsf-training%2Fcpluspluscourse%2Ftree%2Fmaster%2Fcode. Should we use that URL?

Reply:
> Let me find out if you can create stable links (== still work when we replace the PDF with a new version) on Indico.

In case you got the wrong idea: It's about downloading the C++ source files for the exercises, not the course PDF.

Reply:
OK, that's fine!

Reply:
I see. Yes, let them download everything.